src/gui/MItem_menus.cpp

```diff
@@ -402,17 +402,37 @@ MI_LOAD_SETTINGS::MI_LOAD_SETTINGS()
     : WI_LABEL_t(_(label), nullptr, is_enabled_t::yes, is_hidden_t::no) {
 }
 
+struct SettingsLoadState {
+    bool is_network;
+    bool is_connect;
+};
+
 void MI_LOAD_SETTINGS::click(IWindowMenu & /*window_menu*/) {
-    // FIXME: Some error handling/reporting
-    // TODO: Loading other things than just network
+    SettingsLoadState settings_loaded = SettingsLoadState { .is_network = true, .is_connect = true };
```

Comment:
Avoid this initialization by setting default values in the struct definition.

Also, I'd prefer having them initialized to false and not true, doesn't make much sense like this to me.

Reply:
Removed, so no longer relevant